src/applications/disability-benefits/686/config/form.js

```diff
@@ -121,81 +124,74 @@ const formConfig = {
+                claimantSocialSecurityNumber: ssnUI,
+                dateOfBirth: currentOrPastDateUI('Date of birth'),
+                relationshipToVet: {
+                  'ui:title': 'Relationship to Veteran',
+                  'ui:widget': 'radio',
+                  'ui:options': {
+                    labels: {
+                      1: 'I am the Veteran',
+                      2: 'Spouse or surviving spouse',
+                      3: 'Unmarried adult child',
+                      4: 'Other'
+                    },
+                    nestedContent: {
+                      2: spouseRelationshipDescription,
+                      3: childRelationshipDescription,
+                      4: otherRelationshipDescription
+                    }
+                  }
                 },
-                last: {
-                  'ui:required': (formData) => formData['view:relationship'] !== 'veteran'
-                }
-              }),
-              ssn: _.assign(ssnUI, {
-                'ui:required': (formData) => formData['view:relationship'] !== 'veteran'
-              }),
-              address: address.uiSchema('', false, (formData) => {
-                return formData['view:relationship'] !== 'veteran';
-              }),
-              claimantEmail: {
-                'ui:title': 'Email address',
-                'ui:required': (formData) => formData['view:relationship'] !== 'veteran'
-              },
-              'ui:options': {
-                expandUnder: 'view:relationship',
-                expandUnderCondition: (field) => field === 'spouse' || field === 'child' || field === 'other'
               }
-            },
+            }
           },
           schema: {
             type: 'object',
-            required: ['view:relationship'],
             properties: {
-              veteranFullName,
-              veteranSocialSecurityNumber,
-              veteranVAfileNumber: vaFileNumber,
-              'view:relationship': {
-                type: 'string',
-                'enum': [
-                  'veteran',
-                  'spouse',
-                  'child',
-                  'other'
-                ]
-              },
-              'view:applicantInfo': {
+              application: {
                 type: 'object',
                 properties: {
-                  claimantFullName,
-                  ssn,
-                  address: address.schema(fullSchema686),
-                  claimantEmail
+                  claimant: {
+                    type: 'object',
+                    required: [
+                      'claimantFullName',
+                      'claimantSocialSecurityNumber',
+                      'dateOfBirth',
+                      'relationshipToVet'
+                    ],
+                    properties: {
+                      claimantFullName: _.merge(fullName, {
+                        properties: {
+                          maiden: {
```

Comment:
I put a comment on the main issue asking Emily about it, but we probably don't want to add this.